### PR TITLE
buildSpan is no more

### PR DESCRIPTION
**Pull request by beniwohli**
also, set api_request_time to a low value to avoid timeouts

### CODE REVIEW

File: docker/nodejs/express/app.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ var apm = require('elastic-apm-node').start({
   serviceName: process.env.EXPRESS_SERVICE_NAME,
   flushInterval: 1,
   maxQueueSize: 1,
+  api_request_time: "50ms",
```
Comment (watson) [inline docker/nodejs/express/app.js]:
```suggestion
  apiRequestTime: "50ms",
```

File: docker/nodejs/express/app.js [hunk 2]
```diff
@@ -23,8 +24,7 @@ app.get("/foo", function(req, res) {
 });
 
 function foo_route () {
-    var span = apm.buildSpan()
-    span.start('app.foo')
+    var span = apm.startSpan('app.foo', 'custom')
```
Comment (watson) [inline docker/nodejs/express/app.js]: `custom` type is the default, so no need to set it 😃 

```suggestion
    var span = apm.startSpan('app.foo')
```

File: docker/nodejs/express/app.js [hunk 3]
```diff
@@ -34,15 +34,13 @@ app.get("/bar", function(req, res) {
 });
 
 function bar_route () {
-    var span = apm.buildSpan()
-    span.start('app.bar')
+    var span = apm.startSpan('app.bar', 'custom')
```
Comment (watson) [inline docker/nodejs/express/app.js]:
```suggestion
    var span = apm.startSpan('app.bar')
```

File: docker/nodejs/express/app.js [hunk 4]
```diff
@@ -34,15 +34,13 @@ app.get("/bar", function(req, res) {
...
     extra_route()
     span.end()
 }
 
 function extra_route () {
-    var span = apm.buildSpan()
-    span.start('app.extra')
+    var span = apm.startSpan('app.extra', 'custom')
```
Comment (watson) [inline docker/nodejs/express/app.js]:
```suggestion
    var span = apm.startSpan('app.extra')
```

File: docker/nodejs/express/app.js (hunk 1 above)
Comment (beniwohli) [reply docker/nodejs/express/app.js]: oops

File: docker/nodejs/express/app.js (hunk 2 above)
Comment (beniwohli) [reply docker/nodejs/express/app.js]: I was going with "let's be explicit in tests". OK to leave it in?

Comment (watson) [reply docker/nodejs/express/app.js]: Up to you 👍

**Comment by beniwohli** [comment]
didn't fix the tests yet, but cut down the console output from 15mbyte to a few hundred kilobyte, so I'll chalk that up as a success

**Pull request merged**